ctgan/synthesizers/ctgan.py

```diff
@@ -232,6 +233,11 @@ def fit(self, train_data, discrete_columns=tuple()):
                 contain the integer indices of the columns. Otherwise, if it is
                 a ``pandas.DataFrame``, this list should contain the column names.
         """
+        if epochs:
+            warnings.warn("Passing 'epochs' to 'fit' will be deprecated in future versions. "
```

Comment:
I would say that it "has been deprecated and will be removed" instead of saying that it will be removed. Let's also use a `DeprecationWarning`, since this message is intended for developers more than users.